### ISSUE TITLE
.4.1 bot going for same Pokemon after capture

### ISSUE THREAD

**Issue by Spyderdreams** [jabbink/PokemonGoBot#269]
**Description:**
bot going for same Pokemon after capture

**Steps to reproduce:**

1. start bot
2. watch as it captures Pokemon
3. you can restart the bot in place and it will also keep going for same Pokemon that was captured by you.
**Expected behavior:**
It should not be tring to catch a Pokemon already captured

**Actual behavior:**
`24 Jul 22:55:18: XP gain: 0 XP; Pokemon caught/transferred: 0/0; Items caught/dr
opped: 3/3; Pokebank 80/250; Stardust 139900
24 Jul 22:55:22: Getting map of (40.76133445666225, -73.9779996283986)
←[32m24 Jul 22:55:27: Found pokemon KRABBY ←[0m
24 Jul 22:55:27: Getting map of (40.761348151899455, -73.97799837685909)
←[32m24 Jul 22:55:28: Encountered pokemon KRABBY with CP 52 ←[0m
←[33m24 Jul 22:55:28: Using ITEM_POKE_BALL ←[0m
←[32m24 Jul 22:55:30: Found pokemon KRABBY ←[0m
←[31m24 Jul 22:55:31: Encounter failed with result: ENCOUNTER_ALREADY_HAPPENED ←
[0m
24 Jul 22:55:32: Getting map of (40.761324637918236, -73.97798979601727)
←[32m24 Jul 22:55:33: Caught a KRABBY with CP 52 and IV 51%: [200x XP, 3x Candy,
 100x Stardust] ←[0m
←[33m24 Jul 22:55:34: Dropped 1x ITEM_HYPER_POTION ←[0m
←[33m24 Jul 22:55:34: Going to transfer KRABBY with CP 52 and IV 51%; reason: CP
 < 400 and IV < 80% ←[0m
←[32m24 Jul 22:55:36: Found pokemon KRABBY ←[0m
←[32m24 Jul 22:55:36: Found pokemon KRABBY ←[0m
←[33m24 Jul 22:55:37: Going to transfer KRABBY with CP 52 and IV 51%; reason: CP
 < 400 and IV < 80% ←[0m
←[33m24 Jul 22:55:37: Going to transfer KRABBY with CP 52 and IV 51%; reason: CP
 < 400 and IV < 80% ←[0m
24 Jul 22:55:37: Getting map of (40.76141522896804, -73.97805671608592)
←[31m24 Jul 22:55:37: Encounter failed with result: ENCOUNTER_ALREADY_HAPPENED ←
[0m
←[33m24 Jul 22:55:37: Going to transfer KRABBY with CP 52 and IV 51%; reason: CP
 < 400 and IV < 80% ←[0m
←[31m24 Jul 22:55:37: Encounter failed with result: ENCOUNTER_ALREADY_HAPPENED ←
[0m
←[33m24 Jul 22:55:37: Going to transfer KRABBY with CP 52 and IV 51%; reason: CP
 < 400 and IV < 80% ←[0m
←[32m24 Jul 22:55:42: Found pokemon KRABBY ←[0m
24 Jul 22:55:42: Getting map of (40.76140647590481, -73.97800563064622)
←[31m24 Jul 22:55:43: Encounter failed with result: ENCOUNTER_ALREADY_HAPPENED ←
[0m
24 Jul 22:55:44: Walking to pokestop "Lapstrake 1987" (40.761133, -73.978286) in
 62.914984433585005 steps.
24 Jul 22:55:47: Getting map of (40.76133249385731, -73.97810741763156)
←[32m24 Jul 22:55:50: Found pokemon KRABBY ←[0m
←[31m24 Jul 22:55:51: Encounter failed with result: ENCOUNTER_ALREADY_HAPPENED ←`

**Stacktrace (If it's a crash):**
[Please use pastebin if it's too long]

**Version:**
.4.1

**Comment by iamthasasquatch** [comment]
I beleive this means you are at the max amount of pokemon. Not sure if the build he released had a fix for that. I would login manually and see if you are at the 250/250 pokemon limit if so evolve or transfer some

**Comment by jabbink** [comment]
Fixed I think. Can you confirm that?

**Comment by Spyderdreams** [comment]
Yes, it is fixed but I found out something out even better. 

I can capture over the max with a trick i figured out. If you never login to the account the max Pokemon doesn't matter. I got up to over 500 hundred Pokemon on one account I created using PTC. This is a game bug not bot bug.

additionally regular Poki balls will not go to 0 

`Found pokemon RATTATA ←[0m
Encountered pokemon RATTATA with CP 38 ←[0m
Using ITEM_POKE_BALL ←[0m`

item_poke_ball=0
